### DISCUSSION TITLE
VMware - Failed to load ldlinux.c32

### DISCUSSION THREAD

**Discussion by geegol** [Security-Onion-Solutions/securityonion#13140] [Unsupported Versions]
Good evening, 

I just downloaded the iso from the security onion website and tried to spin up a virtual machine with Vmware workstatiion. However, when I spin it up, I get a black screens howing an error "Failed to load ldlinux.c32. Boot failed: press a key to retry" I tried adjusting my RAM. I went into the VMs bios settings to see what might have went wrong. I chose "other linux 32 bit" for the OS type. I don't know what I am doing wrong.

 I tried googling for a guide on how to spin up security onion but I cannot find any guides regarding this error and I also tried searching github and reddit for a guide but could not find any answers to this question. All the guides I found were a step by step on how to setup a VM. Which I know how to do. What can I do to get this working? Is the ISO corrupted? What am I doing wrong? 

Thanks in advance.

**Comment by dougburks** [comment]
> I chose "other linux 32 bit" for the OS type

Please note that Security Onion is a 64-bit operating system. For more information, please see:
https://docs.securityonion.net/en/2.4/trouble-booting.html

> Is the ISO corrupted?

You should be able to verify this as shown at:
https://securityonion.com/download
